docs/reference/modules/piece-page-type.md

```diff
@@ -0,0 +1,240 @@
+---
+extends: '@apostrophecms/doc-type'
+---
+
+# `@apostrophecms/piece-page-type`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module cooperates with the 'piece-type' module to expose two specialized views. The [index page](https://v3.docs.apostrophecms.org/reference/glossary.html#index-page) displays all pieces of a particular `piece-type` in a paginated, filterable manner. The [show page](https://v3.docs.apostrophecms.org/reference/glossary.html#show-page) is for presenting individual pieces. These features are added to those exposed by the  ['@apostrophecms/page'](/reference/modules/page.md) module.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`next`](#next) | Boolean \|\| Object | If set to `true`, `data.next` is the next piece based on the sort. |
+| [`perPage`](#perpage) | Integer | The number of pieces to include in a set of `GET` request results. |
+| [`piecesFilters`](#piecesFilter) | Array | Takes an array of objects where each contains a `name` key and a value of a field in the piece to filter on. |
+| [`pieceModuleName`](#pieceModuleName) | String | Optionally sets the `piece-type` to a specific name other than the default set by the module name. |
+| [`previous`](#previous) | Boolean \|\| Object | If set to `true`, `data.previous` is the previous piece based on the sort. |
+
+### `next`
+If this option is set to true, it exposes the next piece in the current [sort order](https://v3.docs.apostrophecms.org/reference/module-api/module-options.html#sort) as `req.data.next` (`data.next` in the template) when serving a [show page](https://v3.docs.apostrophecms.org/reference/glossary.html#general-terms). This can be used to provide a link to the next item in a series (e.g., the next oldest blog post).
+
+This option can also be set to an object containing an object with a key of `project` and values which will be used as a query builder for retrieving the next piece document.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  extend: ‘@apostrophecms/piece-page-type’,
```

Comment:
Wrong quotes.

Reply:
I should have noticed that. As an aside - I'm not completely positive how I made what I assume is a forward tick.